zebra/kernel_netlink.c

```diff
@@ -1958,6 +1959,9 @@ void kernel_init(struct zebra_ns *zns)
 		       &zns->t_netlink);
 
 	rt_netlink_init();
+
+	if (zns->ge_netlink_cmd.sock >= 0)
```

Comment:
I'd move this if to inside the call to ge_netlink_init().  kernel_init() shouldn't care about the status.  It just knows it needs to ask the question.

Reply:
Moved to `ge_netlink_init()`.